From dccee3bd3c14a6df1e71253dc7f5574a50e170bc Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Fri, 24 Mar 2023 19:21:24 -0700
Subject: [PATCH] Update path to libretro.cpp

---
 bsnes/target-libretro/GNUmakefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bsnes/target-libretro/GNUmakefile b/bsnes/target-libretro/GNUmakefile
index 35e65a4..96c719b 100644
--- a/bsnes/target-libretro/GNUmakefile
+++ b/bsnes/target-libretro/GNUmakefile
@@ -19,7 +19,7 @@ endif
 objects := libretro $(objects)
 objects := $(patsubst %,obj/%.o,$(objects))
 
-obj/libretro.o: target-libretro/libretro.cpp
+obj/libretro.o: libretro.cpp
 
 all: $(objects)
 ifeq ($(platform),linux)
-- 
2.37.0.windows.1

